### PR TITLE
[CK_TILE] Improve headdim96 performance for fmha-bwd

### PR DESCRIPTION
**Pull request by qianfengz**
The codes including some change in ck_tile fmha-bwd,  but mainly tested/verified by xforemers:

- The instances for hd96 is added by xformers,  not added into ck_tile to save compiling-time
- Added hd96 in xformers/ops/fmha/ck.py BwOp to let test_backward to generate cases for hd96

The work brings approximately 15% performance improvement using the benchmark scripts from one important customer

### CODE REVIEW

File: include/ck_tile/ops/fmha/pipeline/tile_fmha_traits.hpp [hunk 1]
```diff
@@ -33,6 +33,30 @@ struct TileFmhaTraits
     static constexpr index_t kBlockPerCu    = kBlockPerCu_;
 };
 
+template <bool kPadSeqLenQ_ /* padding for seqlen_q */,
```
Comment (danyao12) [inline include/ck_tile/ops/fmha/pipeline/tile_fmha_traits.hpp]: What's the the difference between TileFmhaTraits and TileFmhaBwdTraits?

Comment (qianfengz) [reply include/ck_tile/ops/fmha/pipeline/tile_fmha_traits.hpp]: Removed  `TileFmhaBwdTraits`

**Comment by danyao12** [comment]
LGTM

**Review by carlushuang — APPROVED** [review]
LGTM

**Pull request merged**